woocommerce/woocommerce-android
# Use shared debug keystore

This PR updates the build configuration in order to make debug builds use a shared keystore instead of the default one.
It also switches the Installable builds to Wasabi Debug so that this key is the only one required by the CI (this will also be changed soon, as I see that a Jalapeno build is going to be implemented in https://github.com/woocommerce/woocommerce-android/pull/2427). 

To test:
_Wasabi debug build_

1. Checkout this branch.
2. Rename your local default debug keystore (`~/.android/debug.keystore`) so that it can't be found by gradle.
3. Build `WasabiDebug`, run it and verify that Google Login doesn't work.
4. Verify that a local key was not regenerated. 
5. Run `./gradlew installDeps`, verify that your configuration gets updated and that the shared keystore is copied to your `~/.android/ folder`.
6. Build WasabiDebug, run it and verify that Google Login now works.

_Installable builds_
7. Download the installable apk from this PR and verify that it can be installed and that Google Login works.

_Wrap up_:
8. Restore your default keystore.

Note: When everything is ready, before merging this PR, the mobile repository shall be merged and the configuration in this PR updated. 

Update release notes:
